modules/core/test/v2/unit/coins/abstractUtxoCoin.ts

```diff
@@ -418,6 +432,18 @@ describe('Abstract UTXO Coin:', () => {
     }));
 
     it('should construct an unsigned recovery transaction for the offline vault', co(function *() {
+      const callBack1 = sandbox.stub(Btc.prototype, 'getAddressInfoFromExplorer');
+      callBack1.returns(Promise.resolve(emptyAddressInfo));
+      callBack1.withArgs('2N8cRxMypLRN3HV1ub3b9mu1bbBRYA4JTNx').returns(Promise.resolve({txCount: 2, totalBalance: 0}));
+      callBack1.withArgs('2MxZA7JFtNiQrET7JvywDisrZnKPEDAHf49').returns(Promise.resolve({txCount: 2, totalBalance: 100000}));
+      callBack1.withArgs('2MtHCVNaDed65jnq6YUN7qiHoef6xGDH4PR').returns(Promise.resolve({txCount: 2, totalBalance: 0}))
+      callBack1.withArgs('2N6swovegiiYQZpDHR7yYxvoNj8WUBmau3z').returns(Promise.resolve({txCount: 2, totalBalance: 120000}))
+
+      const callBack2 = sandbox.stub(Btc.prototype, 'getUnspentInfoFromExplorer');
+      callBack2.withArgs('2MxZA7JFtNiQrET7JvywDisrZnKPEDAHf49').returns(Promise.resolve([addressUnspents['2MxZA7JFtNiQrET7JvywDisrZnKPEDAHf49']]));
+      callBack2.withArgs('2MtHCVNaDed65jnq6YUN7qiHoef6xGDH4PR').returns(Promise.resolve([addressUnspents['2MtHCVNaDed65jnq6YUN7qiHoef6xGDH4PR']]));
+      callBack2.withArgs('2N6swovegiiYQZpDHR7yYxvoNj8WUBmau3z').returns(Promise.resolve([addressUnspents['2N6swovegiiYQZpDHR7yYxvoNj8WUBmau3z']]));
+      callBack2.returns(Promise.resolve([]));
```

Comment:
these are all testnet address prefixes. we should probably have a test from a real address

https://en.bitcoin.it/wiki/List_of_address_prefixes

p2sh and p2wsh

Reply:
These are just mocked responses anyways, so I am not sure what difference it makes if we use a real address. We are not actually making any calls anyways. Could you elaborate?